lib/developer_portal/lib/liquid/drops/provider.rb

```diff
@@ -201,7 +201,7 @@ def logo_url
 
       desc 'Returns API spec collection.'
       def api_specs
-        Drops::Collection.for_drop(Drops::ApiSpec).new(@model.api_docs_services)
+        Drops::Collection.for_drop(Drops::ApiSpec).new(@model.api_docs_services.accessible)
```

Comment:
This has no test :)

Reply:
It does 😝 
 https://github.com/3scale/porta/pull/62/files/67887ede094015013e1f877257c787726c3372f1#diff-94b55e724331e858e0b967a69b96f022R58

Reply:
Just check `test/unit/liquid/drops/provider_drop_test.rb` 👍

Reply:
Sorry it is! https://github.com/3scale/porta/pull/62/files#diff-94b55e724331e858e0b967a69b96f022R58